src/canvas/Renderer.tsx

```diff
@@ -236,15 +236,17 @@ export class Renderer implements IRenderer {
         });
     };
 
-    private _endTracing = (position: Point): void => {
+    private _endTracing = (position: Point, needToSnapEndPoint: boolean): void => {
```

Comment:
Can we change needToSnapEndPoint to snapEndPoint

Reply:
sure!